libs/NfcCoreLib/lib/LibNfc/phLibNfc_Hci.c

```diff
@@ -1288,7 +1284,47 @@ phLibNfc_HciGetHostListProc(void* pContext,NFCSTATUS status,void* pInfo)
     return wStatus;
 }
 
-static NFCSTATUS phLibNfc_NfceeModeSet(void *pContext,NFCSTATUS wStatus,void *pInfo)
+static NFCSTATUS phLibNfc_HostModeSet(void *pContext, NFCSTATUS wStatus, void *pInfo)
```

Comment:
If pHciContext->bNoOfHostsETSI12Compliant is 0 there is no need to call gphLibNfc_HciChildDevCreateApduPipeInitSequenceNci1x  it will call gphLibNfc_HciEndInitsequenceNci1x.